src/widgets/WidgetMainQmc.vue

```diff
@@ -107,6 +116,95 @@ export default defineComponent({
       return this.cards.slice(0, 4)
     },
   },
+  created() {
+    this.getPrimaryProduct()
+    this.getCardAccountFee()
+    this.getCardStandardInterestRate()
+    this.createTitle()
+    this.createSubtitle()
+  },
+  methods: {
+    getPrimaryProduct() {
+      this.primaryProduct.productType = this.products[0].productType
+      this.primaryProduct.productItems = this.products[0].productItems
+    },
+    getCardAccountFee() {
+      this.cardAccountFee = this.cards.find(
+        card => card.id === 'QMC1'
+      )?.annualFee
+    },
+    getCardStandardInterestRate() {
+      this.cardStandardInterestRate = this.cards.find(
+        card => card.id === 'QMC1'
+      )?.interestRate
+    },
+    getLongTermInterestFreeTitle() {
+      const interestFreePeriod = this.primaryProduct.productItems[0].contents.find(
+        item => item.key === 'interestFreePeriod'
+      )?.value
+      const indicativeMinMonthly = this.primaryProduct.productItems[0].contents.find(
+        item => item.key === 'indicativeMinMonthly'
+      )?.value
+      return `${interestFreePeriod} Month${
+        interestFreePeriod === 1 ? '' : 'ly'
+      } Payment${
+        interestFreePeriod === 1 ? '' : 's'
+      } of $${indicativeMinMonthly}*`
+    },
+    getPaymentHolidayTitle() {
+      const interestFreePeriod = this.primaryProduct.productItems[0].contents.find(
+        item => item.key === 'interestFreePeriod'
+      )?.value
+      return `${interestFreePeriod} Month${
+        interestFreePeriod === 1 ? '' : 's'
+      } No Payments & No Interest`
```

Comment:
I see you're doing a lot of pluralisation of strings. You should probably create a utility function for handling that in a consistent and reusable way